common-service-core/src/test/java/org/zowe/apiml/message/log/ApimlLoggerTest.java

```diff
@@ -37,6 +39,39 @@ void testEmpty() {
         assertNull(apimlLogger.log("someKey"));
     }
 
+    @Test
+    void when_nullMessageService_return_nullMessage() {
+        ApimlLogger apimlLogger = ApimlLogger.empty();
+
+        assertNull(ReflectionTestUtils.getField(apimlLogger, "messageService"));
+        assertNull(apimlLogger.log("org.zowe.apiml.common.invalidMessageKey"));
+    }
+
+    @Test
+    void when_invalidKey_return_invalidKeyMessage() {
```

Comment:
I'm commenting on the names of the tests, generally for new tests we are using the `@Nested` approach, to have inner classes split by given conditions.
This test class is from before that, although maybe it's ok to add a `@Nested` class. You can find an example of this in `CorsUtilsTest`.